### ISSUE TITLE
AOSP 10.0.0_r39 requires edits to compile

### ISSUE THREAD

**Issue by pete4abw** [EssentialOpenSource/device_essential_mata#10]
AOSP/device/essential/mata/overlay/packages/apps/Settings/res/values*/strings.xml defines

`<string name="smart_battery_footer" product="default" >`

But this is defined earlier.
```
FAILED: AAPT2 link /share/software/Android/OUT10/target/product/mata/obj/APPS/Settings_intermediates/package-res.apk
Outputs: /share/software/Android/OUT10/target/product/mata/obj/APPS/Settings_intermediates/package-res.apk /share/software/Android/OUT10/target/common/obj/APPS/Settings_intermediates/aapt2.srcjar
/share/software/Android/OUT10/target/common/obj/APPS/Settings_intermediates/extra_packages /share/software/Android/OUT10/target/common/obj/APPS/Settings_intermediates/src/R.stamp /share/software/A
ndroid/OUT10/target/common/obj/APPS/Settings_intermediates/proguard_options
Error: exited with code: 1
Command: /bin/bash /share/software/Android/OUT10/target/product/mata/obj/APPS/Settings_intermediates/package-res.apk.rsp
Output:
warn: removing resource com.android.settings:string/nfc_osaifu_keitai_lock_lock_summary without required default value.
warn: removing resource com.android.settings:string/nfc_osaifu_keitai_lock_setting_toggle_title without required default value.
warn: removing resource com.android.settings:string/nfc_osaifu_keitai_lock_summary_uim_locked without required default value.
warn: removing resource com.android.settings:string/nfc_osaifu_keitai_lock_summary_uim_not_support without required default value.
warn: removing resource com.android.settings:string/nfc_osaifu_keitai_lock_summary_uim_notting without required default value.
warn: removing resource com.android.settings:string/nfc_osaifu_keitai_lock_unlock_summary without required default value.
warn: removing resource com.android.settings:string/nfc_osaifu_remote_settings_title without required default value.
warn: removing resource com.android.settings:string/nfc_osaifu_settings_title without required default value.
warn: removing resource com.android.settings:string/nfc_payment_setting_ng_button without required default value.
warn: removing resource com.android.settings:string/nfc_payment_setting_ng_summary without required default value.
warn: removing resource com.android.settings:string/reader_writer_p2p_off_summary without required default value.
warn: removing resource com.android.settings:string/reader_writer_p2p_on_summary without required default value.
warn: removing resource com.android.settings:string/reader_writer_p2p_toggle_title without required default value.
device/essential/mata/overlay/packages/apps/Settings/res/values/strings.xml:5: error: multiple default products defined for resource com.android.settings:string/smart_battery_footer.
packages/apps/Settings/res/values/strings.xml:5563: note: default product also defined here.
etc... for all other languages
```
This definition has to be removed for compilation to continue. This error appears to happen after r36.

[strings.diff.gz](https://github.com/EssentialOpenSource/device_essential_mata/files/4736737/strings.diff.gz)

**Comment by jb-essential** [comment]
Thanks a lot for the report

Should be fixed with dfa264f5fb3e04777f299af00308af52d59c495b

**Comment by pete4abw** [comment]
NP. Unfortunately, there may be other issues as the Settings Menu under
`Settings/Network & Internet/Mobile network/Advanced`
There is no choice for Wi-Fi Calling.
Other menus are prefaced with `null`.

Not really usable...

But at least with this change, compile can conclude so further investigation can occur.

FYI: Reverting to stock will result in the red screen "Your Device is Corrupt". I got rid of it with

`adb reboot "dm-verity enforcing"`
